pkg/services/resourcesFilterService.go

```diff
@@ -12,6 +14,38 @@ type RootObject struct {
 }
 
 func ShouldResourceBeValidated(admissionReviewReq *admission.AdmissionReview, rootObject RootObject) bool {
+	isResourcesSkipByConfig := configAllowedListsValidation(admissionReviewReq, rootObject)
+
+	if isResourcesSkipByConfig {
+		return false
+	}
+
+	return defaultResourcesValidation(admissionReviewReq, rootObject)
+}
+
+func configAllowedListsValidation(admissionReviewReq *admission.AdmissionReview, rootObject RootObject) bool {
```

Comment:
I suggest adding a unit test or some comments to explain what's going on here

Reply:
added [https://github.com/datreeio/admission-webhook-datree/pull/148/files#diff-88bac975afdf2a22f485952c5947cd0f154dd016c30fd2ea62f8db2cbc8eb5f7](test)